pep257.py

```diff
@@ -778,6 +778,16 @@ def check_newline_after_last_paragraph(self, definition, docstring):
                                  'quotes on separate line')
 
     @check_for(Definition)
+    def check_surrounding_whitespaces(self, definition, docstring):
+        """D210: White spaces around doctring should be trimmed."""
```

Comment:
You should update the error message here as well (you updated only the README) and in the tests.